evaporate.js

```diff
@@ -597,7 +597,6 @@
                         setStatus(COMPLETE);
                     } else {
                         l.d('headObject not found on S3.');
-                        me.name = awsKey;
```

Comment:
Ah, I think I didn't explain myself properly. I mean that me.name is the file name and in our case awsKey includes a folder (and file name) to upload the file to on S3 so this may have caused the issue, I'm not sure? I just figured if me.name needed to be reset then it would have also been done in the `onErr` handler which we've tested a lot and it works well.